org/mozilla/jss/tests/all.pl

```diff
@@ -455,14 +461,9 @@ sub print_case_result {
 
 my $serverCommand;
 
-$jarFiles = "";
-
-if( $ENV{DEBIAN_BUILD} ) {
-    $jarFiles = "/usr/share/java/slf4j-api.jar:/usr/share/java/slf4j-jdk14.jar:/usr/share/java/commons-lang.jar";
-} else {
-    $jarFiles = "/usr/share/java/slf4j/slf4j-api.jar:/usr/share/java/slf4j/slf4j-jdk14.jar:/usr/share/java/commons-lang.jar";
-}
-
+$jarFiles = Common::get_jar_files;
+$jarFiles = "$jarFiles:/usr/share/java/slf4j/jdk14.jar";
```

Comment:
Perhaps verify this location across Debian and Fedora. I'd like a `get_jar_file` which takes an argument and searches a few locations for the jar in question; should be better for builds.

Reply:
I see ensure_dir_exists so perhaps also ensure_file_exists to check the jar file exists. You'll know how best to do do it.